fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/indexes/RankIndexTest.java

```diff
@@ -1015,10 +1023,12 @@ public void twoRankPredicates() throws Exception {
                         Query.rank(Key.Expressions.field("score").groupBy(Key.Expressions.field("gender"))).equalsValue(1L)))
                 .build();
         RecordQueryPlan plan = planner.plan(query);
-        assertEquals("Index(rank_by_gender [[M, 1],[M, 1]] BY_RANK)" +
-                        " | score LESS_THAN $__rank_0" +
-                        " WHERE __rank_0 = BasicRankedRecord$score.score_for_rank_else_skip(3)",
-                plan.toString());
+        assertThat(plan, scoreForRank(contains(
+                hasToString("__rank_0 = BasicRankedRecord$score.score_for_rank_else_skip(3)")),
+                // need to explicitly construct QueryComponent because we're using an internal parameter
```

Comment:
Maybe a helper here in this fixture for that?

Reply:
Yes, good idea. Fixed.